lib/ClangImporter/ImportType.cpp

```diff
@@ -3224,6 +3224,9 @@ ImportedType ClangImporter::Implementation::importMethodParamsAndReturnType(
           importedType.isImplicitlyUnwrapped()};
 }
 
+ImportedType tryImportOptionsTypeForField(const clang::QualType type,
+                                          ClangImporter::Implementation &Impl);
+
 ImportedType ClangImporter::Implementation::importAccessorParamsAndReturnType(
```

Comment:
You don't have to fix it in this patch, but I wonder if we have the same problem when importing a function that takes the `importMethodParamsAndReturnType` path. Can you try using `SWIFT_COMPUTED_PROPERTY` or `SWIFT_NAME(getter:)` to see if it takes the other path/needs an update?

Reply:
@zoecarver Do you have any specific frameworks like UIKit or CoreGraphics etc in mind I could use to look for one of these?